build.sh

```diff
@@ -9,8 +9,8 @@ PLATFORM=${PLATFORM:-sgx}
 CCF_UNSAFE=${CCF_UNSAFE:-OFF}
 ENABLE_PREFIX_TREE=${ENABLE_PREFIX_TREE:-OFF}
 BUILD_TESTS=${BUILD_TESTS:-ON}
-CC=${CC:-clang-10}
-CXX=${CXX:-clang++-10}
+CC=${CC:-clang-11}
+CXX=${CXX:-clang++-11}
```

Comment:
Would these need to be set to either clang-11 or clang-15 depending on the `PLATFORM` variable? Since clang-11 is supported for SGX and clang-15 is supported for virtual

Reply:
Good point, added a conditional check based on PLATFORM